_releaser/Dockerfile

```diff
@@ -63,6 +63,6 @@ RUN --mount=type=bind,target=. \
   --mount=type=secret,id=AWS_SECRET_ACCESS_KEY \
   --mount=type=secret,id=AWS_SESSION_TOKEN \
   AWS_LAMBDA_FUNCTION_FILE=cloudfront-lambda-redirects.js \
-  REDIRECTS_JSON=$(jq -c '.' /site/redirects.json) \
+  REDIRECTS_JSON=$(jq -c '.' /site/public/redirects.json) \
```

Comment:
I don't think this is the actual issue. It looks like `_redirects.yml` is not handled anymore as it was before in https://github.com/docker/docs/blob/4bbeb2093f9a97b00ced7e8929ce6e9d29a2430a/_plugins/pageless_redirects.rb#L20-L38